.github/workflows/release.yml

```diff
@@ -375,6 +375,7 @@ jobs:
             if [ "${release_required}" == "true" ]; then
               echo "Updating installation script for ${binary}, version: ${version}"
 
+              git checkout master
```

Comment:
aren't we already at master here?

Reply:
Only in the first iteration

Reply:
With https://github.com/calimero-network/core/pull/1089, we'll be releasing them together. We can benefit from using one PR to update both files.